### PR TITLE
Update docs to reflect actual data

### PR DESCRIPTION
**Pull request by ddelnano**
#### :rocket: Why this change?
Seems like the docs are out of sync with the transaction data structure.

#### :memo: Related issues and Pull Requests
https://github.com/snikch/goodman/pull/37

#### :white_check_mark: What didn't I forget?
I still haven't completed verified this is the case since the bug report does not have much detail but skimming the dredd source code I see [this](https://github.com/apiaryio/dredd/blob/acecae523d71b71f47cc66e8587d83e5104ebded/src/transaction-runner.coffee#L310).

- [ ] To write docs
- [ ] To write tests
- [ ] To put [Conventional Changelog](https://dredd.readthedocs.io/en/latest/contributing/#sem-rel) prefixes in front of all my commits and run `npm run lint`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
     "mocha-lcov-reporter": "^1.3.0",
     "nock": "^9.0.13",
     "ps-node": "^0.1.5",
-    "semantic-release": "^7.0.1",
+    "semantic-release": "^7.0.2",
```
Comment (honzajavorek) [inline package.json]: This has been already done in https://github.com/apiaryio/dredd/pull/871/files, but thanks!

**Comment by hobofan** [comment]
@honzajavorek The Rust hooks are fine there. As of now, `Transaction`s are parsed as a pretty loosely structured `JSON` type, so changes to the structure or unknown fields won't easily break it.

File: package.json (hunk 1 above)
Comment (ddelnano) [inline package.json]: Yea sorry my rebase to fix the commit message accidentally picked this up.

**Comment by IngmarStein** [comment]
Thanks for getting this in so fast 💯

**Pull request merged**